provider-ui/package.json

```diff
@@ -36,7 +36,7 @@
     "react-jss": "^8.1.0",
     "react-select": "^2.4.3",
     "styled-jsx": "latest",
-    "uuid": "^3.3.2"
+    "uuid": "^8.1.0"
```

Comment:
I wonder if uuid is used in provider-ui at all; if it is a candidate for removal.

Reply:
Yup, it is a candidate for removal